tests/integration/test_backup_restore_s3/test.py

```diff
@@ -20,13 +23,123 @@
     ],
     with_minio=True,
     with_zookeeper=True,
+    stay_alive=True,
 )
 
 
+def setup_minio_users():
```

Comment:
Can you please add a comment here to describe what are those users `miniorestricted1` and what is the difference between them?